<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>711</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="cpu_info">
     <attribute name="title">
      <string>CPU</string>
     </attribute>
     <widget class="QTableWidget" name="cpu_widget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>691</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mem_info">
     <attribute name="title">
      <string>Memory</string>
     </attribute>
     <widget class="QTableWidget" name="mem_widget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>681</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="curves">
     <attribute name="title">
      <string>Curves</string>
     </attribute>
     <widget class="QCustomPlot" name="cpu_curve" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>661</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="mem_curve" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>661</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>系统：</string>
      </property>
     </widget>
     <widget class="QLabel" name="sys_cpu">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>用户：</string>
      </property>
     </widget>
     <widget class="QLabel" name="usr_cpu">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>total：</string>
      </property>
     </widget>
     <widget class="QLabel" name="nonidle_cpu">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>160</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>线程：</string>
      </property>
     </widget>
     <widget class="QLabel" name="thread_num">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>59</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>进程：</string>
      </property>
     </widget>
     <widget class="QLabel" name="proc_num">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>160</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>物理内存：</string>
      </property>
     </widget>
     <widget class="QLabel" name="mem_total">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>160</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>180</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>已使用：</string>
      </property>
     </widget>
     <widget class="QLabel" name="mem_use">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>180</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>已缓存：</string>
      </property>
     </widget>
     <widget class="QLabel" name="buffer">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>200</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>已交换：</string>
      </property>
     </widget>
     <widget class="QLabel" name="swap">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>220</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <zorder>sys_cpu</zorder>
     <zorder>label_8</zorder>
     <zorder>nonidle_cpu</zorder>
     <zorder>proc_num</zorder>
     <zorder>usr_cpu</zorder>
     <zorder>label_9</zorder>
     <zorder>label_6</zorder>
     <zorder>thread_num</zorder>
     <zorder>label_10</zorder>
     <zorder>label_7</zorder>
     <zorder>cpu_curve</zorder>
     <zorder>mem_curve</zorder>
     <zorder>label_11</zorder>
     <zorder>mem_total</zorder>
     <zorder>label_12</zorder>
     <zorder>mem_use</zorder>
     <zorder>label_13</zorder>
     <zorder>buffer</zorder>
     <zorder>label_14</zorder>
     <zorder>swap</zorder>
    </widget>
    <widget class="QWidget" name="system">
     <attribute name="title">
      <string>System</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>hostname:</string>
      </property>
     </widget>
     <widget class="QLabel" name="host_name">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>system version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="version">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>start time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="start_time">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cur_time">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>累计运行时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="run_time">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="kill_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>25</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>❌</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_edit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
